### PR TITLE
Feat/prepare communication

### PR DESCRIPTION
**Pull request by ahoy-jon**
With giter8 fixed, we can review, fix, then merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +79,15 @@ You can find the latest version on
 ZIO-Spark is compatible with Scala 2.11, 2.12 and 2.13. Spark is provided, you must add your own Spark version in 
 build.sbt (as you would usually). 
 
+```scala
+libraryDependencies ++= Seq(
+  "io.univalence"    %% "zio-spark"  % "0.10.0",
+  "org.apache.spark" %% "spark-core" % "3.3.1" % Provided,
```
Comment (dylandoamaral) [inline README.md:85]: Are you sure about the Provided flag for the user imports ?

File: README.md [hunk 2]
```diff
@@ -107,19 +138,114 @@ scalaVersion := "3.2.1"
...
+|------------------|------------------|
+| sql.Dataset      | sql.Dataset      |
+| sql.SparkSession | sql.SparkSession |
+| ...              | ...              |
+
+
+
+It comes with different API, for example : 
+
+```scala
```
Comment (dylandoamaral) [inline README.md:164]: I am not sure if it is interesting to show the wrapping code; Under the hood we actually call the function below. It is not zio-spark compare to spark but zio-spark wrappring the spark function.

File: docs/experimental/cancellable.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
```
Comment (dylandoamaral) [inline docs/experimental/cancellable.md:1]: Ça c'est très cool, c'est vrai que ça manquait !

File: examples/zparkio/src/main/scala/example2.scala [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+trait Application /*extends ZparkioApp[Arguments, APP_ENV, Unit]*/ {
+  trait APP_ENV
+
+  protected def env: ZLayer[Arguments, Throwable, FileIO with Database] = FileIO.Live ++ Database.Live
+
+  protected def sparkFactory: SparkBuilder.type = SparkBuilder
+  // protected def loggerFactory  = zio.logging
+  // TODO : Show examples with a logger (reusing loggers in Spark with an effect system wrapper)
+
+  protected def makeCli(args: List[String]): Arguments = ??? // Arguments(args)
```
Comment (dylandoamaral) [inline examples/zparkio/src/main/scala/example2.scala:165]: We can replace it with zio config to show that zio-spark is well integrated into the zio ecosystem

Comment (ahoy-jon) [reply examples/zparkio/src/main/scala/example2.scala:165]: yep, we need an example with zio-cli + zio-config and proper logging ( #288 )

File: README.md (hunk 1 above)
Comment (ahoy-jon) [reply README.md:85]: yes, that what we should to be compatible with a spark-submit.

**Pull request merged**